internal/commands/app/create.go

```diff
@@ -65,19 +65,18 @@ func (cmd *CommandCreate) Handler(profile *user.Profile, ui terminal.UI, clients
 	}
 
 	var groupID = cmd.inputs.Project
```

Comment:
[drive-by] not sure why we used this syntax, but could you change this to `groupID := cmd.inputs.Project`?

---

alternatively, we could just mutate `cmd.inputs.Project` itself here and not have a `groupID` variable at all.  if you go that route, then i'd also make the (aesthetic) change to L#70 below and rename `id` to `groupID`